outlines/routing.md

```diff
@@ -0,0 +1,40 @@
+# Routing
+
+* What is routing, and how it's different in a client-rendered app
+    * Uses for the URL
+        * Bookmarking
+        * Sharing
+        * Back/forward
+    * It's a useful serialization of some share-able, bookmarkable client-side state
+        * Not everything needs to be a route - consider that you can just store JavaScript variables, or use local storage for temporary information
+    * Routing is not related to data loading or authorization in a client-side app. For those, go see the security and data loading chapters
+* Flow Router intro
+* Creating routes
+    * How to define a basic route
+    * How to accept route parameters and URL pattern matching
+    * When to use query parameters vs. path parameters
+* Getting information about the current route
```

Comment:
Where do we encourage people to do this. I would say only:
 1. In the components/templates that are rendered by the router (i.e. the page and it's layouts).

[In Galaxy we settled on a pattern where the `GalaxyApp` component (which renders the current page in it's layout(s)) passes `params` in a props to each of the things it renders (the 1. above). So no other component ever accesses `FlowRouter` directly for data. I like this pattern but it doesn't fit with vanilla FR.]